### PR TITLE
Add DAL

### PR DESCRIPTION
**Pull request by Stupid-Fox**

### CODE REVIEW

File: CoffeeHouse.DAL/Repository/CoffeeRepository.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿
```
Comment (hromau) [inline CoffeeHouse.DAL/Repository/CoffeeRepository.cs]: it's looks awful

File: CoffeeHouse.DAL/Repository/Interface/ICoffeeRepository.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CoffeeHouse.DAL.Controllers
+{
+    public interface ICoffeeRepository 
+    {
+
```
Comment (hromau) [inline CoffeeHouse.DAL/Repository/Interface/ICoffeeRepository.cs]: look

File: CoffeeHouse.DAL/Repository/CoffeeRepository.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+using System.Threading.Tasks;
+
+namespace CoffeeHouse.DAL.Repository
+{
+
+    public class CoffeeRepository : ICoffeeRepository
+    {
+        private readonly CoffeeContext _context;
+
+
```
Comment (hromau) [inline CoffeeHouse.DAL/Repository/CoffeeRepository.cs:19]: here is extra spaces

**Review by hromau — APPROVED** [review]

**Comment by hromau** [comment]
why witout implementaion?

**Review by stas1408123 — CHANGES_REQUESTED** [review]
Implement the Repository

**Review by hromau — APPROVED** [review]

**Review by stas1408123 — APPROVED** [review]

**Pull request merged**